src/plugins/pcr_primer_design_for_dna_assembly/src/tasks/FindPresenceOfUnwantedParametersTask.cpp

```diff
@@ -30,16 +32,55 @@ FindPresenceOfUnwantedParametersTask::FindPresenceOfUnwantedParametersTask(const
       settings(_settings) {}
 
 void FindPresenceOfUnwantedParametersTask::run() {
-    //TODO
+    if (settings.bachbone5Length < 0 || settings.bachbone3Length < 0) {
+        setError("Backbone length must be greater than 0");
+        return;
+    }
+    if (sequence.length() < settings.bachbone5Length) {
+        stateInfo.addWarning("Sequence length is less than backbone5Length, the entire sequence is used");
```

Comment:
`backbone5Length` looks like a variable name, it could be replaced with "5' backbone length" - like it's texted in the tab